### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by zhengqixi**

### CODE REVIEW

File: fuzzer/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+To build the `Fuzzy Wuzzy` executable, simply run `go build`. To build the module images, run `bash build_images.sh`.
+
+# Usage 
+
+Once built, `Fuzzy Wuzzy` is invoked like any other executable. 
+By default, `Go` places the executable in the same folder from which
+`go build` was invoked, with the executable taking the name of the the containing folder.
+
+
+`Fuzzy Wuzzy` only takes one argument, a config file. THe config file itself has two sections:
```
Comment (reggieag) [inline fuzzer/README.md:12]:
```suggestion
`Fuzzy Wuzzy` only takes one argument, a config file. The config file itself has two sections:
```
nit, typo

File: fuzzer/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Build
+
+To build the `Fuzzy Wuzzy` executable, simply run `go build`. To build the module images, run `bash build_images.sh`.
+
+# Usage 
+
+Once built, `Fuzzy Wuzzy` is invoked like any other executable. 
+By default, `Go` places the executable in the same folder from which
+`go build` was invoked, with the executable taking the name of the the containing folder.
```
Comment (reggieag) [inline fuzzer/README.md:9]: How about adding a concrete example of what to run in bash after building?

Comment (zhengqixi) [reply fuzzer/README.md:9]: Good call

**Review by reggieag — APPROVED** [review]
LGTM

**Pull request merged**